include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -57,6 +57,86 @@ template <int dim,
 class FEEvaluation;
 
 
+/**
+ * @brief The EvaluationFlags enum
+ */
+namespace EvaluationFlags
+{
+  /**
+   * @brief The type enum
+   */
+  enum type
```

Comment:
Maybe call it `EvaluationFlags` as well? Also, the documentation above shows up in doxygen but doesn't give anything away if you miss the name of the surrounding namespace. I think you might want to say a couple more words there :-)

Reply:
hm, that makes it ``EvaluationFlags::EvaluationFlags`` in the function args, which is a bit weird. Do we have an example somewhere in the library where we wrap an enum like this?